Source/Scene/Cesium3DTile.js

```diff
@@ -396,26 +421,28 @@ define([
         },
 
         /**
-         * Whether the computedTransform has changed this frame.
-         *
-         * @memberof Cesium3DTile.prototype
-         *
-         * @type {Boolean}
          * @readonly
+         * @private
          */
-        transformDirty : {
+        requestServer : {
             get : function() {
-                return this._transformDirty;
+                return this._requestServer;
             }
         },
 
         /**
+         * Determines if the tile has available content to render.  <code>true</code> if the tile's
+         * content is ready or if it has expired content that renders while new content loads; otherwise,
+         * <code>false</code>.
+         *
+         * @memberof Cesium3DTile.prototype
+         *
+         * @type {Boolean}
          * @readonly
-         * @private
          */
-        requestServer : {
+        contentAvailable : {
             get : function() {
-                return this._requestServer;
+                return this.contentReady || (defined(this._expiredContent) && this._contentState !== Cesium3DTileContentState.FAILED);
```

Comment:
This could be tweaked depending on the behavior we want. If a request fails do we want the tile to not be rendered, or do we want to render the expired content? This is doing the former.